src/main/java/com/sparrowwallet/sparrow/io/satochip/Constants.java

```diff
@@ -0,0 +1,149 @@
+    ///** DEPRECATED - Entered PIN is not correct */
+    //public final static short SW_AUTH_FAILED = (short) 0x9C02;
+    /** Required operation is not allowed in actual circumstances */
+    public final static short SW_OPERATION_NOT_ALLOWED = (short) 0x9C03;
+    /** Required setup is not not done */
+    public final static short SW_SETUP_NOT_DONE = (short) 0x9C04;
+    /** Required setup is already done */
+    public final static short SW_SETUP_ALREADY_DONE = (short) 0x9C07;
+    /** Required feature is not (yet) supported */
+    final static short SW_UNSUPPORTED_FEATURE = (short) 0x9C05;
+    /** Required operation was not authorized because of a lack of privileges */
+    public final static short SW_UNAUTHORIZED = (short) 0x9C06;
+    /** Algorithm specified is not correct */
+    public final static short SW_INCORRECT_ALG = (short) 0x9C09;
+
+    /** There have been memory problems on the card */
+    public final static short SW_NO_MEMORY_LEFT = (short) 0x9C01;
+    ///** DEPRECATED - Required object is missing */
+    //public final static short SW_OBJECT_NOT_FOUND= (short) 0x9C07;
+
+    /** Incorrect P1 parameter */
+    public final static short SW_INCORRECT_P1 = (short) 0x9C10;
+    /** Incorrect P2 parameter */
+    public final static short SW_INCORRECT_P2 = (short) 0x9C11;
+    /** Invalid input parameter to command */
+    public final static short SW_INVALID_PARAMETER = (short) 0x9C0F;
+
+    /** Eckeys initialized */
+    public final static short SW_ECKEYS_INITIALIZED_KEY = (short) 0x9C1A;
+
+    /** Verify operation detected an invalid signature */
+    public final static short SW_SIGNATURE_INVALID = (short) 0x9C0B;
+    /** Operation has been blocked for security reason */
+    public final static short SW_IDENTITY_BLOCKED = (short) 0x9C0C;
+    /** For debugging purposes */
+    public final static short SW_INTERNAL_ERROR = (short) 0x9CFF;
+    /** Very low probability error */
+    public final static short SW_BIP32_DERIVATION_ERROR = (short) 0x9C0E;
+    /** Incorrect initialization of method */
+    public final static short SW_INCORRECT_INITIALIZATION = (short) 0x9C13;
+    /** Bip32 seed is not initialized*/
+    public final static short SW_BIP32_UNINITIALIZED_SEED = (short) 0x9C14;
+    /** Bip32 seed is already initialized (must be reset before change)*/
+    public final static short SW_BIP32_INITIALIZED_SEED = (short) 0x9C17;
+    //** DEPRECATED - Bip32 authentikey pubkey is not initialized*/
+    //public final static short SW_BIP32_UNINITIALIZED_AUTHENTIKEY_PUBKEY= (short) 0x9C16;
+    /** Incorrect transaction hash */
+    public final static short SW_INCORRECT_TXHASH = (short) 0x9C15;
+
+    /** 2FA already initialized*/
+    public final static short SW_2FA_INITIALIZED_KEY = (short) 0x9C18;
+    /** 2FA uninitialized*/
+    public final static short SW_2FA_UNINITIALIZED_KEY = (short) 0x9C19;
+
+    /** HMAC errors */
+    static final short SW_HMAC_UNSUPPORTED_KEYSIZE = (short) 0x9c1E;
```

Comment:
Is package visibility wanted? If yes maybe reoredring them will make it more readable